<?xml version="1.0"?>
<launch>
  <arg name="calibration" default="" />
  <node pkg="but_velodyne_proc" type="laser_scan" name="but_velodyne_proc_laserscan_node" output="screen">
    <remap from="points_in" to="/vehicle/velodyne_points" />
    <remap from="scan_out" to="/velodyne/scan" />
    <param name="min_z" value="0.1" />
    <param name="max_z" value="100.0" />
    <param name="angular_res" value="1.5" />
    <param name="min_range" value="1.2" />
    <!--param name="frame_id" value="/velodyne" /-->
  </node>
</launch>
